backend/src/api/organizations.ts

```diff
@@ -64,8 +62,9 @@ class NewOrganizationNonGlobalAdmins {
   @IsBoolean()
   isPassive: boolean;
 
-  @IsBoolean()
-  inviteOnly: boolean;
+  @IsUUID()
+  @IsOptional()
+  parentOrganization?: string;
```

Comment:
I think we should move this property to `NewOrganization` instead, since we currently don't do any validation on `parentOrganization`.